@3rdweb-sdk/react/cache-keys.ts

```diff
@@ -9,28 +9,29 @@ export const networkKeys = {
 
 export const accountKeys = {
   all: ["account"] as const,
-  wallet: (walletAddress: string) =>
+  wallet: (walletAddress: string | undefined) =>
     [...accountKeys.all, walletAddress] as const,
-  me: (walletAddress: string) =>
+  me: (walletAddress: string | undefined) =>
     [...accountKeys.wallet(walletAddress), "me"] as const,
-  usage: (walletAddress: string) =>
+  usage: (walletAddress: string | undefined) =>
     [...accountKeys.wallet(walletAddress), "usage"] as const,
 };
 
 export const apiKeys = {
   all: ["api"] as const,
-  wallet: (walletAddress: string) => [...apiKeys.all, walletAddress] as const,
-  keys: (walletAddress: string) =>
+  wallet: (walletAddress: string | undefined) =>
+    [...apiKeys.all, walletAddress] as const,
+  keys: (walletAddress: string | undefined) =>
     [...apiKeys.wallet(walletAddress), "keys"] as const,
-  key: (id: string, walletAddress: string) =>
+  key: (id: string, walletAddress: string | undefined) =>
     [...apiKeys.keys(walletAddress), id] as const,
 };
 
 export const authorizedWallets = {
   all: ["authorizedWallets"] as const,
-  wallet: (walletAddress: string) =>
+  wallet: (walletAddress: string | undefined) =>
```

Comment:
instead of doing this, add an invariant

  invariant(address, "Address is not provided");